base/daemon/SentryWriter.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <common/types.h>
+#include <Common/StackTrace.h>
+
+#include <Poco/Util/LayeredConfiguration.h>
+
+#include <string>
+
+/// Sends crash reports to ClickHouse core developer team via https://sentry.io
+class SentryWriter
+{
+public:
+    SentryWriter() = delete;
+
+    static void initialize(Poco::Util::LayeredConfiguration & config);
+    static void shutdown();
+    static void onFault(
```

Comment:
Need to write in comments that this method is not signal safe and cannot be called from inside signal handler.